Update Consumption Unit to Watt-hour to Match In-Car Unit

mattw01 opened the pull request:
Updated Consumption unit in Drives and Efficiency dashboards to Watt-hour (Wh) per #675

adriankumpf commented on grafana/dashboards/drives.json:
```diff
@@ -578,7 +578,7 @@
           "metricColumn": "none",
           "rawQuery": true,
-          "rawSql": "WITH data AS (\n  SELECT\n    round(extract(epoch FROM start_date)) * 1000 AS start_date_ts,\n    round(extract(epoch FROM end_date)) * 1000 AS end_date_ts,\n    car.id as car_id,\n    CASE WHEN start_geofence.id IS NULL THEN CONCAT('new?lat=', start_position.latitude, '&lng=', start_position.longitude)\n         WHEN start_geofence.id IS NOT NULL THEN CONCAT(start_geofence.i ...
+          "rawSql": "WITH data AS (\n  SELECT\n    round(extract(epoch FROM start_date)) * 1000 AS start_date_ts,\n    round(extract(epoch FROM end_date)) * 1000 AS end_date_ts,\n    car.id as car_id,\n    CASE WHEN start_geofence.id IS NULL THEN CONCAT('new?lat=', start_position.latitude, '&lng=', start_position.longitude)\n         WHEN start_geofence.id IS NOT NULL THEN CONCAT(start_geofence.i ...
```
In 563cf7832a2845daaa17d810924989140df32154 the `reduced_range` field was slightly modified. These changes are not reflected here. Could you please rebase/merge your branch to pull in this commit?

adriankumpf replied:
The Trip dashboard has a minimal "Drives" panel that shares most fields with this table. I think it makes sense to update the consumption there as well.

mattw01 replied:
Ah, I was testing with the last release version and forgot to merge in the latest commits. Thanks for catching that. Also updated the Trip dashboard as well.

adriankumpf commented on grafana/dashboards/efficiency.json:
```diff
@@ -181,8 +181,8 @@
-      "decimals": 1,
-      "format": "kwatth",
+      "decimals": 0,
+      "format": "watth",
```
The `/ 100km` (`/ 100mi`) postfix value also needs to be changed for both consumption panels :)

mattw01 replied:
Fixed these panels, thanks!

mattw01 commented:
Modified it a bit more to include the length unit.
Grafana supports custom units since version 6.6. I updated the units to include the current $length_unit. So it should display either Wh/km or Wh/mi to be a bit more specific.

adriankumpf commented on grafana/dashboards/efficiency.json:
```diff
@@ -477,12 +477,12 @@
           "thresholds": [],
           "type": "number",
-          "unit": "kwatth"
+          "unit": "Wh/km"
```
Nice! 👍
